eberron.md

```diff
@@ -201,7 +210,7 @@ The restriction on the wood's weight is increased by 5 pounds for every level ca
 - **Range:** 10
 - **Duration:** 10 hours
 
-Causes a spectral steed to form in an area that you can see. The steed may be a horse or other non-violent creature of large size or smaller.
+Causes a spectral steed to form in an area that you can see. The steed may be a horse or other non-violent creature of large size or smaller. Make a spellcasting roll against 2 bad dice. On success the steed is summoned.
```

Comment:
Adding a small difficulty to the spell so the same reasons as other spells: Anyone with a codex and the spell can cast it, not just people proficient in that school.